pylint/extensions/_check_docs_utils.py

```diff
@@ -41,6 +41,21 @@ def returns_something(return_node):
     return not (isinstance(returns, astroid.Const) and returns.value is None)
 
 
+def inside_yields(node):
```

Comment:
Minor point: Please could you rename this to something like `inside_generator` or `has_child_yield`. This is because, to me, "`inside_yields`" sound like the method would be checking if the given node is part of a statement for a yield (ie that a yield is a parent node).